### PR TITLE
started two cube auto on aligned scale

### PR DESCRIPTION
**Pull request by christiancruzfhs**

### CODE REVIEW

File: Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        // brings wrist to pickup position and elevator down
+        this.addParallel(setWristToPickUp, 1);
+        setElevatorForPickup.changeTimeout(3.5);
+        this.addSequential(setElevatorForPickup);
+        
+        // go to first cube on ground position
+        this.addSequential(firstCubeFromGround);
+        
+        // intake block
+        this.addSequential(intake);
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java]: intake should probably be in parallel with driving towards the cube, and also it has to end after some time

File: Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+        // Get in position
+        this.addSequential(pursuit);
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java]: We need a new movement to get back to to scoring position

**Review by JohnGilb — CHANGES_REQUESTED** [review]

File: Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java (hunk 1 above)
Comment (Gabriel-J-Young) [reply Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java]: done

File: Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java (hunk 2 above)
Comment (Gabriel-J-Young) [reply Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java]: done

**Comment by JohnGilb** [comment]
Integrated manually in other branches.

**Pull request closed without merging**